src/pages/Aboutpage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import AOS from 'aos';
+import 'aos/dist/aos.css';
+import TeamMember from '../components/TeamMember';
+import './Aboutpage/Aboutpage.css';
+const Aboutpage = () => {
+  useEffect(() => {
+    AOS.init({
+      duration: 2000,
+    });
+  });
+  return (
+    <>
+      <div
```

Comment:
can we add bubbles in the background here?
you can ask Hewr about it or Abdulrahman they have images of bubbles